specify_manual/templates/plan-template.md

```
@@ -101,4 +101,4 @@ directories captured above]
 | Violation | Why Needed | Simpler Alternative Rejected Because |
 |-----------|------------|-------------------------------------|
 | [e.g., 4th project] | [current need] | [why 3 projects insufficient] |
-| [e.g., Repository pattern] | [specific problem] | [why direct DB access insufficient] |
+| [e.g., Repository pattern] | [specific problem] | [why direct DB access insufficient] |
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)